spec/decorators/course_decorator_spec.rb

```diff
@@ -243,6 +243,12 @@
 
         it { is_expected.to eq("Student finance if you’re eligible") }
       end
+
+      context "Courses excluded from bursaries" do
+        let(:course) { build :course, name: "Physics with mathematics" }
```

Comment:
https://find-postgraduate-teacher-training.education.gov.uk/course/B72/F3GX

Reply:
Ah my boolean is round the wrong way ... will fix